darts/timeseries.py

```diff
@@ -725,14 +740,15 @@ def from_series(pd_series: pd.Series,
         TimeSeries
             A TimeSeries constructed from the inputs.
```

Comment:
You should also add the documentation for dummy_index here.

Reply:
Done